engine/src/sol/commitment_config.rs

```diff
@@ -0,0 +1,195 @@
+#![allow(deprecated)]
```

Comment:
@albert-llimos I'm assuming this is copied code, right? Could you comment that on the files that are copied, and putting them together seems like a good idea.